packages/mobile/src/redux/migrations.ts

```diff
@@ -621,4 +621,5 @@ export const migrations = {
       ..._.omit(state.walletConnect, ['v2']),
     },
   }),
+  51: (state: any) => state,
```

Comment:
i was being lazy 🙈 when you add a property, this is enough because the state reconciler will pick it up and merge the new initial state with the persisted one. it's nice to be specific about what you're adding for transparency but we also do this in the `schema` file now. we technically only need to be specific if we are removing or renaming/restructuring properties :)